packages/components/src/modifiers/hds-advanced-table-cell/dom-management.ts

```diff
@@ -35,9 +34,7 @@ export const updateTabbableChildren = (
 };
 
 export const didInsertGridCell = (
-  cell:
-    | HdsAdvancedTableThSignature['Element']
-    | HdsAdvancedTableTdSignature['Element']
+  cell: HdsAdvancedTableThSignature['Element']
```

Comment:
Was this changed because the ThSignature and TdSignature are both HTMLDivElement?

Reply:
missed this one! @aklkv was this intentional? otherwise we can raise a follow-up to bring it back

Reply:
yeah it was implying identical value
<img width="1146" alt="Screenshot 2025-04-02 at 4 35 17 PM" src="https://github.com/user-attachments/assets/b3e38c18-35b2-4407-be1c-29ef23acb6d8" />

Reply:
Figured, that should be fine. Those types will always be the same.